File: templates/python/hvdc_facts_state_map.py
"""hvdc_facts_state_map: HVDC 与 FACTS状态热力图（heatmap 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='HVDC and FACTS analysis: state heatmap'):
    return make_template_figure('heatmap', seed=3703, title=title, domain='HVDC and FACTS analysis', topic='state heatmap')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/hvdc_facts_state_map.m
function fig = hvdc_facts_state_map()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('heatmap', 3703, 'HVDC and FACTS analysis: state heatmap', 'HVDC and FACTS analysis', 'state heatmap');
end
